### PR TITLE
Added a Pre-conversion Quickstart

### PR DESCRIPTION
**Pull request by arburka**
Adding a quickstart for CentOS Linux pre-conversion analysis

### CODE REVIEW

File: docs/quickstarts/insights-tasks-pre-conversion/insights-tasks-pre-conversion.yml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        Before running the pre-conversion analysis in Red Hat Insights, you must connect your CentOS machines to Insights<br>  
+        
+        **To verify CentOS Linux machines are connected:**
+
+        1. Connect CentOS Linux machines to Insights.
+          <details>
+            <summary><span style="color:#0066CC">&nbsp**Need to connect a CentOS Linux machine?**</span></summary>
+            - Install the Red Hat client tools:
+                - Download the Red Hat GPG key:<br>
+                `# curl -o /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release https://www.redhat.com/security/data/fd431d51.txt`{{copy}}<br>
```
Comment (mshriver) [inline docs/quickstarts/insights-tasks-pre-conversion/insights-tasks-pre-conversion.yml:46]: `sudo` is necessary here, I'd rather that then assume or instruct to run directly as root.

**Comment by arburka** [comment]
@mshriver @Hyperkid123 I added a note in the quickstart - please let me know if it resolves the issue!

File: docs/quickstarts/insights-tasks-pre-conversion/insights-tasks-pre-conversion.yml (hunk 1 above)
Comment (mshriver) [reply docs/quickstarts/insights-tasks-pre-conversion/insights-tasks-pre-conversion.yml:47]: Above comment added on L44 provides the necessary context, thanks!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**